### ISSUE TITLE
Can't create project using Compass v0.13.alpha.0

### ISSUE THREAD

**Issue by drejohnson** [at-import/toolkit#1]
this is the error I get in terminal

```
/Users/schapman/.rvm/gems/ruby-1.9.3-p194/gems/bundler-1.1.5/lib/bundler/shared_helpers.rb:22:in `default_gemfile': Could not locate Gemfile (Bundler::GemfileNotFound)
    from /Users/schapman/.rvm/gems/ruby-1.9.3-p194/gems/bundler-1.1.5/lib/bundler.rb:213:in `default_gemfile'
    from /Users/schapman/.rvm/gems/ruby-1.9.3-p194/gems/bundler-1.1.5/lib/bundler.rb:164:in `root'
    from /Users/schapman/.rvm/gems/ruby-1.9.3-p194/gems/bundler-1.1.5/lib/bundler.rb:123:in `load'
    from /Users/schapman/.rvm/gems/ruby-1.9.3-p194/gems/bundler-1.1.5/lib/bundler.rb:114:in `setup'
    from /Users/schapman/.rvm/gems/ruby-1.9.3-p194/gems/bundler-1.1.5/lib/bundler.rb:119:in `require'
    from /Users/schapman/.rvm/gems/ruby-1.9.3-p194/gems/compass-0.13.alpha.0/bin/compass:25:in `<top (required)>'
    from /Users/schapman/.rvm/gems/ruby-1.9.3-p194/bin/compass:19:in `load'
    from /Users/schapman/.rvm/gems/ruby-1.9.3-p194/bin/compass:19:in `<main>'
    from /Users/schapman/.rvm/gems/ruby-1.9.3-p194/bin/ruby_noexec_wrapper:14:in `eval'
    from /Users/schapman/.rvm/gems/ruby-1.9.3-p194/bin/ruby_noexec_wrapper:14:in `<main>'
```

**Comment by drejohnson** [comment]
may be a issue with Compass v0.13.alpha.0 more then Toolkit

**Comment by Snugug** [comment]
So, from the error you've put up here, it actually sounds like an rvm configuration issue as opposed to either a Toolkit or a Compass issue. In fact, someone recently had a similar issue with another project I work on and that turned to be an RVM issue.

I just installed the latest Toolkit from RubyGems, have Compass 0.13.alpha.0 on my computer, and the following command runs fine;

`compass create test -r toolkit --using toolkit/susy`

This furthers my suspicion.

I did a little digging, and it appears to be a bug in Compass related to Bundler. Check out [this thread](http://stackoverflow.com/questions/10659662/compass-gem-throwing-error) for what may be a quick solve and [this thread](http://stackoverflow.com/questions/10610254/cant-install-compass-via-rvm) for an explanation as to what's potentially going wrong.
